### DISCUSSION TITLE
sb is not recognised

### DISCUSSION THREAD

**Discussion by tallevy22** [seleniumbase/SeleniumBase#1059] [Q&A] [answered]
hi i get an error:  missing 1 required positional argument: 'sb'
how do i add the sb fixture?
thanks Tal

**Comment by mdmintz** [comment, marked as answer]
Hi Tal, there are a few different examples that use the ``sb`` fixture:
https://github.com/seleniumbase/SeleniumBase/blob/master/examples/test_sb_fixture.py
https://github.com/seleniumbase/SeleniumBase/blob/master/examples/boilerplates/sb_fixture_test.py
https://github.com/seleniumbase/SeleniumBase/blob/master/examples/boilerplates/samples/sb_swag_test.py
